projects/dotcms-field-elements/src/components/dot-label/dot-label.e2e.ts

```diff
@@ -0,0 +1,166 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
```

Comment:
something missing here ? textarea tests in the dot-label.e2e.ts ?

Reply:
Copy paste.